src/wmnds/components/form-elements/radios/_example.njk

```diff
@@ -0,0 +1,54 @@
+{% raw %}
+{% from "wmnds/components/form-elements/radios/_radios.njk" import wmndsRadios %}
+
+{{
+  wmndsRadios({
+    items: [
+      {
+        id: null,
+        contentText: "Option 1",
+        contentHTML: null,
+        value: "option1",
+        checked: false,
+        name: null,
+        disabled: false
+      },
+      {
+        id: null,
+        contentText: "Option 2",
+        contentHTML: null,
+        value: "option2",
+        checked: true,
+        name: null,
+        disabled: false
+      },
+      {
+        id: null,
+        contentText: "Option 3",
+        contentHTML: null,
+        value: "option3",
+        checked: false,
+        name: null,
+        disabled: false
+      }
+    ],
+    name: "radios-example",
+    hint: {
+      id: null,
+      classes: null,
+      contentText: "Select one option",
+      contentHTML: null
+    },
+    idPrefix: "radios",
+    inline: false,
+    classes: null,
+    error: false,
+    errorMessage: {
+      contentText: "Please select an option",
+      contentHTML: null,
+      classes: null,
+      id: null
```

Comment:
Make sure these are following the same order as the njk properties.
I'd move contentText/HTML to the top as they are required.